src/js/components/Carousel/Carousel.js

```diff
@@ -147,7 +147,8 @@ const Carousel = ({
     if (index === activeIndex) {
       if (priorActiveIndex !== undefined) {
         animation = {
-          type: priorActiveIndex < activeIndex ? 'slideLeft' : 'slideRight',
+          type:
+            play || priorActiveIndex < activeIndex ? 'slideLeft' : 'slideRight',
```

Comment:
Just wondering why we would only tie this to `play`? I would think that `play` just means automatically advance. So, I would think we could just make this "slideRight".

Reply:
I think when `play={false}` and you are manually clicking through the buttons on the left or right to change the image it makes sense if you click the left button to have the animation 'slideLeft' instead of always being 'slideRight'

Reply:
Ah, makes sense. Thanks.